core/src/main/java/io/questdb/griffin/engine/functions/table/ReadParquetRecordCursor.java

```diff
@@ -209,13 +210,13 @@ private boolean symbolToVarcharRemappingDetected(int metadataType, int decoderTy
     }
 
     private class ParquetRecord implements Record {
-        private final DirectBinarySequence binarySequence = new DirectBinarySequence();
-        private final DirectString directCharSequenceA = new DirectString();
-        private final DirectString directCharSequenceB = new DirectString();
-        private final Long256Impl long256A = new Long256Impl();
-        private final Long256Impl long256B = new Long256Impl();
-        private final Utf8SplitString utf8SplitViewA = new Utf8SplitString();
-        private final Utf8SplitString utf8SplitViewB = new Utf8SplitString();
+        private final ObjList<DirectBinarySequence> bsViews = new ObjList<>();
```

Comment:
nit: Can we pass the column count here to determine the exact List sizes we need to create?

Reply:
Applied the suggestion where appropriate in 75f285745923b65824fd8ce16287a7ee9c2ec4ed